i18n/i18n.go

```diff
@@ -56,7 +56,7 @@
 package i18n
 
 import (
-	"github.com/nicksnyder/go-i18n/i18n/bundle"
+	"github.com/dennisfaust/go-i18n/i18n/bundle"
```

Comment:
TravisCI runs this within `$GOPATH/src/github.com/nicksnyder/go-i18n`, so this is actually unnecessary for CI purposes here.